src/PerfectGym.AutomergeBot/RepositoryConnection/RepositoryConnectionContext.cs

```diff
@@ -178,12 +178,26 @@ public IReadOnlyList<PullRequest> GetOpenPullRequests()
             var request = new PullRequestRequest

```

Comment:
Warto przy okazji dodać brakujący log:
```suggestion
             _logger.LogDebug("Getting open pull requests", pushInfoHeadCommitSha);
            var request = new PullRequestRequest
```